### DISCUSSION TITLE
Train Epoch Q&A

### DISCUSSION THREAD

**Discussion by Amorparty** [minsangit/Transfomer#1] [Q&A] [answered]
1. Train Epoch 에 따른 Accuracy 증가?
2. mecab 활용

**Comment by mssongit** [comment, marked as answer]
1. 간단한 예제로 구현해서 Train Epoch를 20회까지만 진행했고, 20회까지는 Accuracy가 계속 상승함.
2. mecab 활용하여 Sentencepiece Vocab을 구현하는 부분은 코드 따로 업로드 예정.
